draft-ietf-oauth-v2-1.md

```diff
@@ -2901,7 +2901,8 @@ practice to steal resource owners' passwords.
 
 Service providers should attempt to educate end users about the risks
 phishing attacks pose and should provide mechanisms that make it easy
-for end users to confirm the authenticity of their sites.  Client
+for end users to confirm the authenticity of their sites, such as using
+phishing-resistant authenticators. Client
```

Comment:
It might not be obvious to anyone why a phishing resistent authenticator makes it easier for a user to confirm a site's authenticity. Something along the lines of "as phishing resistant authentications will only offer a credential to login to a certain site to the user if the platform has successfully authenticated this site."
 
Also Nat's original comment was related to "steal resource owners' password" and suggested to make it more general by referring to pwasswords and other non-phishing resistent credentials (like OTPs).